src/style.css

```diff
@@ -0,0 +1,72 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 12px;
+}
+
+.search-bar-small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  font-style: normal;
+  font-weight: normal;
+  background: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  background-size: 11px;
+}
+
+.search-bar-big {
+  height: 70px ;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background: #fff;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+}
+
+.search-bar {
+  margin-bottom: 20px;
+  width: 100%;
+  font-family: "Avenir";
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-shadow: 1px 8px, rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar:hover {
+  box-shadow: 3px 8px, rgba(61, 78, 97, 0.2);
+}
+
+.search-bar-small:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  background-size: 11px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-bar-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px 25px;
+  background-size: 19px;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
there is no need to add the same styles to focused state, it inherits all styles from normal state